client/components/swipeable/index.js

```diff
@@ -1,42 +1,31 @@
 import classnames from 'classnames';
 import { useRtl } from 'i18n-calypso';
-import { Children, useState, useEffect, useRef, useCallback } from 'react';
+import { Children, useState, useLayoutEffect, useRef, useCallback } from 'react';
 
 import './style.scss';
 
 const OFFSET_THRESHOLD_PERCENTAGE = 0.45; // Percentage of width to travel before we trigger the slider to move to the desired slide.
 const VELOCITY_THRESHOLD = 0.5; // Speed of drag above, before we trigger the slider to move to the desired slide.
+const TRANSITION_DURATION = `300ms`;
```

Comment:
Nit: no need for this to be in a string literal; it can just be a simple string.

Reply:
fixed